charts/shoot-core/components/charts/kube-proxy/templates/psp/kube-proxy-init-script.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-proxy-init-script
+  namespace: kube-system
+data:
+  init.sh: |
+    #!/bin/sh
+    OLD_KUBE_PROXY_MODE=$(cat /kube-proxy-mode);
```

Comment:
As far as I can see this file only exists inside the ephemeral storage of the container. Shouldn't it be on the host/VM itself, i.e., shouldn't you mount a host path where it can be stored?